tests/framework/framework.go

```diff
@@ -83,27 +84,27 @@ func NewFramework(prefix string) *Framework {
 	// handle run-time flags
 	if !flag.Parsed() {
 		flag.Parse()
-		// report flags values passed to test binary
-		fmt.Fprintf(GinkgoWriter, "** Test flags:\n")
-		flag.Visit(func(f *flag.Flag) {
-			fmt.Fprintf(GinkgoWriter, "   %s = %q\n", f.Name, f.Value.String())
-		})
-		fmt.Fprintf(GinkgoWriter, "**\n")
-		if kubectlPath != nil {
-			f.KubectlPath = *kubectlPath
-		}
-		if ocPath != nil {
-			f.OcPath = *ocPath
-		}
-		if cdiInstallNs != nil {
-			f.CdiInstallNs = *cdiInstallNs
-		}
-		if kubeConfig != nil {
-			f.KubeConfig = *kubeConfig
-		}
-		if master != nil {
-			f.Master = *master
-		}
+	}
+	// report flags values passed to test binary
+	fmt.Fprintf(GinkgoWriter, "** Test flags:\n")
+	flag.Visit(func(f *flag.Flag) {
+		fmt.Fprintf(GinkgoWriter, "   %s = %q\n", f.Name, f.Value.String())
+	})
+	fmt.Fprintf(GinkgoWriter, "**\n")
+	if kubectlPath != nil {
+		f.KubectlPath = *kubectlPath
+	}
+	if ocPath != nil {
+		f.OcPath = *ocPath
+	}
+	if cdiInstallNs != nil {
+		f.CdiInstallNs = *cdiInstallNs
+	}
+	if kubeConfig != nil {
+		f.KubeConfig = *kubeConfig
+	}
+	if master != nil {
+		f.Master = *master
```

Comment:
thanks for formatting. I ran `gofmt -w -s` a few times but must have missed this...

Reply:
Actually that was a bug, we create the Framework struct in each high level test file, but we only parse the flags once, so we would only copy the values into the Framework struct the first time. So notice I changed the if to only be around parse.

Reply:
per mention from @copejon none of the _if flag == nil {_ checks are necessary and the code is cleaner w/o them.

Reply:
Sounds good, let me change that.